CreeDictionary/API/models.py

```diff
@@ -548,7 +537,7 @@ def get_preverbs_from_head_breakdown(
             results = []
             for tag in head_breakdown:
 
-                result: Optional[Preverb] = None
+                preverb_result: Optional[Preverb] = None
                 if tag.startswith("PV/"):
                     # use altlabel.tsv to figure out the preverb
 
```

Comment:
I should have mentioned this way but this code is kind of weird with its usage of `altlabel.tsv` to figure out the normative form of the preverb 😬 

There isn't anything to address now, but I think it can and should be done better in the future.